### ISSUE TITLE
Actively change a table-wide `aria-live` element for complete control

### ISSUE THREAD

**Issue by haxxmaxx** [qlik-oss/sn-table#338]
Currently we are listening to selection and focus changes, then changing the aria-live element, which is attached to the table body. This is limiting us in some cases, where we want to convey information that cannot be done (or complicated) with the correct markup. Examples of these are #287, #314, #315 and #331.

Instead of listening to changes, we update aria-live when we actually make them. add/remove rows from selection, move focus, sort, change page, confirm/cancel selection. This gives us full control of what is said and hopefully place nice with the react update flow.

This also means moving the aria-live element outside the table body and table, more or less placing it on the highest level.
